rules/decoder-events.rules

```diff
@@ -71,6 +71,7 @@ alert pkthdr any any -> any any (msg:"SURICATA UDP header length too small"; dec
 alert pkthdr any any -> any any (msg:"SURICATA UDP invalid length field in the header"; decode-event:udp.len_invalid; classtype:protocol-command-decode; sid:2200120; rev:2;)
 alert pkthdr any any -> any any (msg:"SURICATA SLL packet too small"; decode-event:sll.pkt_too_small; classtype:protocol-command-decode; sid:2200041; rev:2;)
 alert pkthdr any any -> any any (msg:"SURICATA Ethernet packet too small"; decode-event:ethernet.pkt_too_small; classtype:protocol-command-decode; sid:2200042; rev:2;)
+alert pkthdr any any -> any any (msg:"SURICATA Ehtertype Unknown"; decode-event:ethernet.unknown_ethertype; classtype:protocol-command-decode; sid:2200121; rev:1;)
```

Comment:
so say we have unknown ethernet type, and it's running at a 100k packets/s. How many minutes before the post-processing pipeline catches fire?

Seriously though, it seems we'd need to threshold this, but don't have many options. by_src/by_dst/by_flow are all unavailable, so only by_rule could work.

Reply:
https://redmine.openinfosecfoundation.org/issues/7132

Reply:
The expressed need was to catch all the packets with unknown ethernet type (and record them in a pcap) not to threshold to know there are some

Reply:
My point here is not about the use case, but more about general self protection. If someone enables this rule, and it is enabled by default in the PR, it can potentially lead to a alert flood if the unsupported type is common on the monitored network.

Reply:
The same is true for many existing rules, hence the ticket.

Reply:
I'll add a threshold for this rule.

Reply:
Now that we use a general thash for the thresholds, a by_srcmac / by_dstmac options shouldn't be hard to add

Reply:
This to me still seems like the best way to threshold here.